bin/gitlab_changelog_generator

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env ruby
```

Comment:
```suggestion
#!/usr/bin/env ruby
```

Reply:
I found this here, and made a repair note: https://github.com/github-changelog-generator/github-changelog-generator/pull/706